### DISCUSSION TITLE
Isn't this repo open for hacktoberfest?

### DISCUSSION THREAD

**Discussion by ArnavK-09** [web-infra-dev/rspress#1467] [General]
?

**Comment by chenjiahan** [comment]
Sure, I have added the `hacktoberfest` topic to this repo, PR welcome!

Contributing guide: https://github.com/web-infra-dev/rspress/blob/main/CONTRIBUTING.md

**Reply by ArnavK-09** [reply]
Thanks, I'll send some prs soon 🎉
